### DISCUSSION TITLE
Question regarding "Film/Video & Live Events" Projects

### DISCUSSION THREAD

**Discussion by Olympusmonsgames** [microdee/OpenUnrealSpaceMouse#79] [Q&A]
It seems that 3dConnexion has an issue controlling viewport camera classes with the 6-DOF Controller Cap on Spacemouse products when UE projects are created using any of the 'Film/Video & Live Events' templates with their official plugin. This has been a known issue since at least Dec 2021, and I can confirm functionality is still broken. All buttons on my SM Enterprise work, I just can't control the viewport camera at all using the Cap.

Does anyone know if this plugin works with UE projects created using those templates?

**3dConnexion forum posts regarding this issue here:** 
[https://forum.3dconnexion.com/viewtopic.php?f=25&t=43030](https://forum.3dconnexion.com/viewtopic.php?f=25&t=43030) (Issue mentioned initially)
&
[https://forum.3dconnexion.com/viewtopic.php?f=25&t=49647](https://forum.3dconnexion.com/viewtopic.php?f=25&t=49647) (Known Issue mentioned again 6-months later)

**Comment by microdee** [comment]
this plugin doesn't use 3DConnexion's library, it gets hardware data directly from HID and implements navigation schemes itself in Unreal, SpaceMouse Enterprise included. You can try it if you download the plugin from the releases https://github.com/microdee/UE4-SpaceMouse/releases/latest

I haven't tried those templates with space mice but I give my plugin 85% chance having no trouble with them
